sources/Cargo.toml

```diff
@@ -81,7 +81,7 @@ members = [
 ]
 
 [workspace.dependencies]
-apiclient = { version = "0.1", path = "api/apiclient" }
+apiclient = { version = "0.1", path = "api/apiclient", default-features = false }
```

Comment:
nit: I don't understand the "enforce TLS when FIPS mode is on" part of the commit message, that should be reworded.